content/change/educate/first-steps.md

```diff
@@ -1,5 +1,7 @@
+title: First steps
```

Comment:
I think these are no longer strictly necessary, as page titles are set in the `mkdocs.yaml` file in the new `mkdocs` version.

Reply:
a'ight!